### PR TITLE
Fix latency benchmarks after introduction of minUTxOValue in tests

### PR DESCRIPTION
**Pull request by piotr-iohk**
# Issue Number

https://github.com/input-output-hk/cardano-wallet/issues/2145

# Overview

- eaa2b0fd7dbaff846f222c584f83b23a2862137f
  Fix latency benchmarks after introduction of minUTxOValue in tests
  
- 943030c81106c8fd6601256d0af035fb2d63f2ee
  use maxDaedalusGap in WALLETS_CREATE_08 - address_pool_gap
  

# Comments

I break and fix.

### CODE REVIEW

**Review by KtorZ — APPROVED** [review]

**Comment by KtorZ** [comment]
> I break and fix.

Move fast, break things. 

Not sure I am fond of this mantra though :sweat_smile:

**Review by paweljakubas — APPROVED** [review]

File: lib/core-integration/src/Test/Integration/Scenario/API/Shelley/Wallets.hs [hunk 1]
```diff
@@ -454,18 +454,25 @@ spec = describe "SHELLEY_WALLETS" $ do
 
     describe "WALLETS_CREATE_08 - address_pool_gap" $ do
         let addrPoolMin = fromIntegral @_ @Int $ getAddressPoolGap minBound
-        let addrPoolMax = fromIntegral @_ @Int $ getAddressPoolGap maxBound
+        -- Although maxAddressPoolGap = 100k, there are performance issues when
+        -- creating/using such wallets. Therefore using max gap that is used in Daedalus.
+        -- Performance issues are addressed in https://jira.iohk.io/browse/ADP-442
+        let maxDaedalusGap = (1_000 :: Word32)
```
Comment (piotr-iohk) [inline lib/core-integration/src/Test/Integration/Scenario/API/Shelley/Wallets.hs:461]: ok?

**Pull request merged**